eos/effects/fighterabilityevasivemaneuvers.py

```diff
@@ -0,0 +1,24 @@
+# Not used by any item
+"""
+Since fighter abilities do not have any sort of item entity in the EVE database, we must derive the abilities from the
+effects, and thus this effect file contains some custom information useful only to fighters.
+"""
+# User-friendly name for the ability
+displayName = "Evasive Maneuvers"
+
+prefix = "fighterAbilityEvasiveManeuvers"
+
+# Is ability applied to the fighter squad as a whole, or per fighter?
+grouped = True
+
+type = "active"
+runTime = "late"
+def handler(fit, module, context):
```

Comment:
Minor quibble, but it should probably be:
`+def handler(fit, container, context):`

Module implies it's actually a module, when it's not, it's a fighter.

I _would_ say use `fighter` instead, but unfortunately we can have effects that cross boundaries, so might as well standardize on something fairly generic.

It's not a major issue, lots of other effect files don't do this, but I've started doing this when creating new effect files (@blitzmann and I had a discussion about it after I submitted code doing the same thing :) ).